node/node.d.ts

```diff
@@ -33,6 +47,7 @@ interface WeakSetConstructor {}
 ************************************************/
 declare var process: NodeJS.Process;
 declare var global: NodeJS.Global;
+declare var console: Console;
```

Comment:
This is going to conflict with lib.d.ta if you are not using TS2.0; I would put this on types-2.0 for now and use @types/node